File: problem_1/solution_1.py
def RemainderIsZero(dividend, divisor):
    return dividend % divisor == 0

def OutputSum():
    sum = 0
    for i in range(1, 1000):
        if(RemainderIsZero(i, 3) or RemainderIsZero(i, 5)):
            sum = sum + i
    print(sum)

OutputSum()
        

File: problem_1/solution1.clj
(defn is-rem-zero?
	[dividend divisor]
	(zero? (rem dividend divisor)))

(defn multiple-of-5?
	[n]
	(is-rem-zero? n 5))

(defn multiple-of-3?
	[n]
	(is-rem-zero? n 3))

(defn multiple-of-3-or-5?
	[n]
	(or (multiple-of-3? n) (multiple-of-5? n)))

(defn solution-1
	[n]
 	(reduce + (filter multiple-of-3-or-5? (range n))))